python/ql/test/experimental/query-tests/Security/CWE-522/ldap2_private.py

```diff
@@ -0,0 +1,66 @@
+import ldap
+from flask import request, Flask
+
+app = Flask(__name__)
+
+schema = "ldap://"
+remote_host = "127.0.0.1"
+full_host = "ldap://127.0.0.1"
+
+
+# GOOD
+
+# SSL through ldap variable option
+# host from variable + variable
+
+@app.route("/one")
+def one():
+    ldap_connection_1 = ldap.initialize(schema + remote_host)
+    ldap_connection_1.set_option(ldap.OPT_X_TLS_DEMAND, True)
```

Comment:
Oh I understood it the whole opposite way. So how should tests be written? Setting a global option will affect the entire file :/

Reply:
Right, that's actually my bad. I found the wrong documentation :sweat_smile: 

The docs I found are for `ldap.set_option` (which sets options globally), and not `LDAPObject.set_option` ([docs](https://www.python-ldap.org/en/python-ldap-3.3.0/reference/ldap.html#ldap.LDAPObject.set_option)).

So this code looks fine to me. And my mistake just highlights that it's possible to make _all_ new LDAP connections secure by using the global `ldap.set_option` (is at least my understanding). It would be nice to have such an example added, if you care to :blush: